config/defaults.go

```diff
@@ -0,0 +1,95 @@
+/*
+   Copyright The Soci Snapshotter Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+/*
+   Copyright 2019 The Go Authors. All rights reserved.
+   Use of this source code is governed by a BSD-style
+   license that can be found in the NOTICE.md file.
+*/
+
+package config
+
+// Config (root) defaults
+const (
+	defaultMetricsNetwork = "tcp"
+)
+
+// ServiceConfig defaults
+const (
+	defaultImageServiceAddress = "/run/containerd/containerd.sock"
+)
+
+// FSConfig defaults
+const (
+	defaultFuseTimeoutSec = 1
+	// Amount of time the background fetcher will wait once a new layer comes in
+	// before (re)starting fetches.
+	defaultBgSilencePeriodMsec = 30_000
+
+	// Specifies how often the fetch will occur.
+	// The background fetcher will fetch a single span every `defaultFetchPeriod`.
+	defaultBgFetchPeriodMsec = 500
+
+	// Specifies the maximum size of the bg-fetcher work queue i.e., the maximum number
+	// of span managers that can be queued. In case of overflow, the `Add` call
+	// will block until a span manager is removed from the workqueue.
+	defaultBgMaxQueueSize = 100
+
+	// The default amount of interval at which the background fetcher emits metrics
+	defaultBgMetricEmitPeriodSec = 10
+
+	// Amount of time Mount will time out if a layer can't be resolved.
+	defaultMountTimeoutSec = 30
+
+	// Amount of time the snapshotter will wait before emitting the metrics for FUSE operation.
+	defaultFuseMetricsEmitWaitDurationSec = 60
+
+	defaultValidIntervalSec = 60
+
+	defaultFetchTimeoutSec = 300
+
+	// DefaultDialTimeoutMsec is the default number of milliseconds before timeout while connecting to a remote endpoint. See `TimeoutConfig.DialTimeout`.
```

Comment:
nit: doc comment name doesn't match variable name since it's not exported any more.